xml/ha_yast_cluster.xml

```diff
@@ -1027,14 +1027,14 @@ Finished with 1 errors.</screen>
   </sect1>
 
   <sect1 xml:id="sec-ha-installation-start">
-   <title>Bringing the Cluster Online</title>
+   <title>Bringing the Cluster Online at the Node Level</title>
```

Comment:
Initially, I did not get what 'at node level' is supposed to mean, I first needed to look into the bug and the plans for the future as outlined there by Roger. How about 'Bringing the Cluster Online Node by Node'? We can also check with Liam (whom I suggest for additional review anyway) which title would be more clear.

Reply:
I prefer @taroth21's title here.

Reply:
Ok, added Tanja's suggestion. Thanks! :+1: